internal/handlers/handler_sign_duo.go

```diff
@@ -21,11 +22,15 @@ func SecondFactorDuoPost(duoAPI duo.API) middlewares.RequestHandler {
 		}
 
 		userSession := ctx.GetSession()
+		log := logging.Logger()
+		remoteIP := ctx.RemoteIP().String()
+
+		log.Debugf("Starting 2FA Duo Push Auth Attempt for %s from IP %s", userSession.Username, remoteIP)
```

Comment:
since you have an AutheliaCtx at hand you can use the dedicated request logger like here: https://github.com/authelia/authelia/blob/a268a16c6e2cd91bbd3513d2851047d3cc6b8cbc/internal/handlers/handler_firstfactor.go#L45

Reply:
Done!